src/main/java/com/nike/cerberus/domain/input/EnvironmentConfig.java

```diff
@@ -61,60 +70,76 @@ public void setEnvironmentName(String environmentName) {
         this.environmentName = environmentName;
     }
 
-    public String getRegion() {
-        return region;
+    public String getPrimaryRegion() {
+        return primaryRegion;
     }
 
-    public void setRegion(String region) {
-        this.region = region;
+    public void setPrimaryRegion(String primaryRegion) {
+        this.primaryRegion = primaryRegion;
     }
 
-    public String getCostCenter() {
-        return costCenter;
+    public Map<String, String> getGlobalTags() {
+        return globalTags;
     }
 
-    public void setCostCenter(String costCenter) {
-        this.costCenter = costCenter;
+    public void setGlobalTags(Map<String, String> globalTags) {
+        this.globalTags = globalTags;
     }
 
-    public String getOwnerEmail() {
-        return ownerEmail;
+    public String getAdminRoleArn() {
+        return adminRoleArn;
     }
 
-    public void setOwnerEmail(String ownerEmail) {
-        this.ownerEmail = ownerEmail;
+    public void setAdminRoleArn(String adminRoleArn) {
+        this.adminRoleArn = adminRoleArn;
     }
 
-    public String getOwnerGroup() {
-        return ownerGroup;
+    public String getBaseDomainName() {
+        return baseDomainName;
     }
 
-    public void setOwnerGroup(String ownerGroup) {
-        this.ownerGroup = ownerGroup;
+    public void setBaseDomainName(String baseDomainName) {
```

Comment:
Javadoc could explain difference between a baseDomainName and an Edge domain overrride, etc.